Gemfile

```diff
@@ -5,5 +5,6 @@ source 'https://rubygems.org'
 gem 'xcpretty', '0.3.0'
 gem 'fastlane', '2.205.1'
 gem 'jazzy', '0.15.1'
+gem 'rexml', '3.3.9'
```

Comment:
Doing this will fail the Dependency Check because it pulls a newer `json` version that has an incompatible license. 

Instead, just run `sudo bundle update rexml` in both locations.